alicloud/resource_alicloud_alikafka_sasl_user.go

```diff
@@ -0,0 +1,147 @@
+package alicloud
+
+import (
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/alikafka"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func resourceAlicloudAlikafkaSaslUser() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAlicloudAlikafkaSaslUserCreate,
+		Read:   resourceAlicloudAlikafkaSaslUserRead,
+		Delete: resourceAlicloudAlikafkaSaslUserDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"instance_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"username": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringLenBetween(1, 64),
+			},
+			"password": {
```

Comment:
please add kms_password to support setting a encrypted password. Please refer to alicloud_db_account or alicloud_instance